pom.xml

```diff
@@ -3085,6 +3085,31 @@
         <orc.classifier>nohive</orc.classifier>
         <datanucleus-core.version>3.2.10</datanucleus-core.version>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>enforce-hadoop-versions</id>
+                <goals>
+                  <goal>enforce</goal>
+                </goals>
+                <configuration>
+                  <rules>
+                    <bannedDependencies>
+                      <excludes>
+                        <exclude>org.apache.hadoop:*:[3.2.0,)</exclude>
```

Comment:
Does this work in `sbt` run? It seems that PR builder is banned by this.
```
[info] Building Spark using SBT with these arguments:  -Phadoop-2.7 -Phive-1.2 -Pspark-ganglia-lgpl -Pyarn -Pkubernetes -Phadoop-cloud -Phive-thriftserver -Phive -Pkinesis-asl -Pmesos test:package streaming-kinesis-asl-assembly/assembly
```

Reply:
It does not work for `sbt` runner, and I don't know how to enable this for it. Any advice？

Reply:
Let's just drop this PR then. We'll deprecate and remove Hive 1.2 profile in the near future anyway.

Reply:
LGTM, I'll close this